chapters/ch04.md

```diff
@@ -69,9 +99,13 @@ Many other details such as neural network architecture are also important, and t
 
 ->
 
+[1] Hình vẽ thể hiện mạng neural cho kết quả tốt hơn trong điều kiện dữ liệu nhỏ. Nhận định này ít đồng thuận hơn với nhận định mạng neural với dữ liệu lớn.  Đối với dữ liệu nhỏ, tùy thuộc vào cách tạo đặc trưng thủ công mà thuật toán truyền thống có thể hoặc không cho kết quả tốt hơn. Ví dụ như, nếu ta chỉ có 20 mẫu huấn luyện thì không đủ để dùng hồi suy logistic lẫn mạng neural. Nhưng đổi lại đặc trưng thủ công lại chịu ảnh hưởng nhiều từ thuật toán học máy. Nhưng nếu có 1 triệu mẫu thì tôi sẽ chọn mạng neural. 
```

Comment:
* "Hình vẽ thể hiện mạng neural cho kết quả tốt hơn trong điều kiện dữ liệu nhỏ. Nhận định này ít đồng thuận hơn với nhận định mạng neural với dữ liệu lớn. " maybe có thể viết như này 

> Mặc dù hình vẽ thể hiện mạng neural cho kết quả tốt hơn trong điều kiện dữ liệu nhỏ. Nhưng nhận định này nhận được ít sự đồng thuận hơn so với nhận định "Mạng neural cho kết quả tốt hơn rõ rệt với tập dữ liệu lớn".

* "Ví dụ như, nếu ta chỉ có 20 mẫu huấn luyện thì không đủ để dùng hồi suy logistic lẫn mạng neural. Nhưng đổi lại đặc trưng thủ công lại chịu ảnh hưởng nhiều từ thuật toán học máy." nên là:

> Ví dụ như, nếu bạn chỉ có 20 mẫu huấn luyện thì gần như việc bạn sử dụng hồi quy logistic hay mạng neural không quan trọng bằng kỹ thuật mà bạn trích xuất đặc trưng như thế nào.

Reply:
@tiepvupsu Có thể dịch là "kỹ thuật trích xuất đặc trưng thủ công" được ko?

Reply:
Mình thấy ok. Từ 'hand-engineering' cũng được đề xuất dịch như thế tại https://github.com/aivivn/Machine-Learning-Yearning-Vietnamese-Translation/pull/87/files.

Reply:
> Mặc dù hình vẽ cho thấy mạng neural cho kết quả tốt hơn với dữ liệu nhỏ, nhận định này ít đồng thuận hơn với nhận định khi dùng mạng neural với dữ liệu lớn. Vì đối với dữ liệu nhỏ, chất lượng thuật toán truyền thống có thể hoặc không cho kết quả tốt hơn mạng neural phụ thuộc rất nhiều vào cách tạo đặc trưng thủ công.

> Và nếu ta chỉ có 20 mẫu huấn luyện thì thực không đủ để dùng cả hồi suy logistic lẫn mạng neural. Nhưng đổi lại, đặc trưng thủ công lại phụ thuộc nhiều vào thuật toán học máy. Nhưng nếu có 1 triệu mẫu, thì tôi sẽ chọn dùng mạng neural.